onnx/helper.py

```diff
@@ -1098,3 +1095,121 @@ def make_training_info(
             binding.value = v
 
     return training_info
+
+
+# Following functions are used for mapping
+def tensor_dtype_to_np_dtype(tensor_dtype: int) -> np.dtype:
+    """
+    Convert a TensorProto's data_type to corresponding numpy dtype. It can be used while making tensor.
+
+    :param tensor_dtype: TensorProto's data_type
+    :return: numpy's data_type
+    """
+    return mapping.TENSOR_TYPE_MAP[int(tensor_dtype)].np_dtype
+
+
+def tensor_dtype_to_storage_tensor_dtype(tensor_dtype: int) -> int:
+    """
+    Convert a TensorProto's data_type to corresponding data_type for storage. It can be used for tensor_dtype_to_storage_numpy_type.
+
+    :param tensor_dtype: TensorProto's data_type
+    :return: data_type for storage
+    """
+    return mapping.TENSOR_TYPE_MAP[tensor_dtype].storage_dtype
+
+
+def tensor_dtype_to_string(tensor_dtype: int) -> str:
+    """
+    Get the name of given TensorProto's data_type.
+
+    :param tensor_dtype: TensorProto's data_type
+    :return: the name of data_type
+    """
+    return mapping.TENSOR_TYPE_MAP[int(tensor_dtype)].name
+
+
+def tensor_dtype_to_storage_numpy_type(tensor_dtype: int) -> np.dtype:
```

Comment:
I wonder if we can omit this function. Users can call the two functions themselves. It is not easy to document/describe, and too many such functions in the API can confuse a reader.

Reply:
Makes sense to me. Just omitted.